src/github.js

```diff
@@ -964,6 +964,22 @@
                body: comment
             }, cb);
          };
+         this.edit = function (issue,options,cb) {
+             if (typeof issue === 'object') {
+                 issue = issue.number;
+             };
+             _request('PATCH', path + '/' + issue, options, function(err, res) {
+               cb(err, res);
+            });
+         };
+         this.get = function (issue,cb) {
+             if (typeof issue === 'object') {
```

Comment:
There is no need to accept an object here. The method retrieves an issue by its number, so the should accept a number as its first argument.